backend/src/main/java/com/bakdata/conquery/resources/api/AutocompleteResource.java

```diff
@@ -0,0 +1,61 @@
+package com.bakdata.conquery.resources.api;
+
+import static com.bakdata.conquery.resources.ResourceConstants.FILTER;
+
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+
+import com.bakdata.conquery.io.jersey.ExtraMimeTypes;
+import com.bakdata.conquery.models.auth.permissions.Ability;
+import com.bakdata.conquery.models.datasets.concepts.Searchable;
+import com.bakdata.conquery.models.datasets.concepts.filters.specific.SelectFilter;
+import com.bakdata.conquery.resources.hierarchies.HAuthorized;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Setter
+@Produces({ExtraMimeTypes.JSON_STRING, ExtraMimeTypes.SMILE_STRING})
+@Consumes({ExtraMimeTypes.JSON_STRING, ExtraMimeTypes.SMILE_STRING})
+// It allows for more than filters but that's not interesting from an API perspective so we keep it at /filters/
+@Path("filters/{" + FILTER + "}/autocomplete")
+@RequiredArgsConstructor(onConstructor_ = {@Inject})
+@ToString
+public class AutocompleteResource extends HAuthorized {
+	@ToString.Exclude
+	private final ConceptsProcessor processor;
+
+	@PathParam(FILTER)
+	protected Searchable<?> searchable;
+
+	@POST
+	public ConceptsProcessor.AutoCompleteResult autocompleteTextFilter(@Valid AutocompleteRequest request) {
+		subject.isPermitted(searchable.getDataset(), Ability.READ);
+
+
+		if (searchable instanceof SelectFilter selectFilter) {
+			subject.isPermitted(selectFilter.getConnector().findConcept(), Ability.READ);
+		}
```

Comment:
Searchable kann auch ein FilterTemplate, welcher dann auch einem Concept in einer oder anderer Weise untergeordnet ist.
Puih, hier haben wir echt einen Bruch mit den Permissions. 
Ein Weg wäre von Concept Permissions wegzukommen und zu Table/ColumnPermissions überzugehen.
Ein anderer Weg wäre das du von dem Searchable auf ein Concept zurückverfolgen kannst, über das der Nutzer Zugriff auf das Searchable hat

Reply:
jep, aber sobald es einem Konzept zugeordnet ist, das für den Nutzer freigegeben ist, dürfte er es momentan benutzen dürfen.